walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,33 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
+    Collection<TimeRange> availibility;
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+
+    //no attendees -- first test
+    if (request.getAttendees() == null){
+        availibility = Arrays.asList(TimeRange.WHOLE_DAY);
+    }
+
+    //no options too long of a request (how do I access time range here? THis is an error) -- second test
+    else if (request.getDuration() > TimeRange.WHOLE_DAY.duration()){
+        availibility = Arrays.asList();
```

Comment:
Should need to do the `asList` conversion here since you don't have a list to convert.  Maybe `new ArrayList<>()` would work as well.

Reply:
> Should need to do the `asList` conversion here since you don't have a list to convert. Maybe `new ArrayList<>()` would work as well.

Sorry typo on the previous comment.  I meant "should**n't** need to do the asList"